bot/exts/evergreen/realpython.py

```diff
@@ -0,0 +1,79 @@
+# Credit to Vthechamp22, who heavily inspired this bit of code
+import logging
+from html import unescape
+from urllib.parse import quote_plus
+
+from discord import Embed, HTTPException
+from discord.ext import commands
+
+from bot import bot
+from bot.constants import Colours
+
+logger = logging.getLogger(__name__)
+
+BASE_URL = "https://realpython.com/search/api/v1/?"
+URL = "https://realpython.com"
+
+ERR_EMBED = Embed(
+    title="Error while searching Real Python",
+    description="There was an error while trying to reach Real Python. Please try again shortly."
+)
+
+
+class RealPython(commands.Cog):
+    """User initiated command to search for a Real Python article."""
+
+    def __init__(self, bot: bot.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["rp"])
+    @commands.cooldown(1, 10, commands.cooldowns.BucketType.user)
+    async def realpython(self, ctx: commands.Context, *, user_search: str) -> None:
+        """Sends 5 articles that match the user's search terms."""
+        params = {"q": user_search}
+        async with self.bot.http_session.get(url=BASE_URL, params=params) as response:
+            if response.status == 200:
+                data = await response.json()
+            else:
+                logger.error(f"Status code is not 200, it is {response.status}")
+                await ctx.send(embed=ERR_EMBED)
+                return
+
+        if len(data["results"]) == 0:
+            no_articles = Embed(
+                title=f"No articles found for {user_search}",
+                color=Colours.soft_red
+            )
+            await ctx.send(embed=no_articles)
+            return
+
+        articles = data["results"][:5]
```

Comment:
Instead of chopping off 5 articles, you could add a `limit` parameter to the URL, so that the API returns at most 5 articles.